### ISSUE TITLE
Random seeds variance

### ISSUE THREAD

**Issue by baize97** [Visual-Attention-Network/SegNeXt#48]
When I try the SegNeXt training with different random seeds, I find that the mIoU will have a large variance (up to 1% to 2%).

Tiny
![image](https://user-images.githubusercontent.com/125171826/218292906-2e61d28d-bd50-4882-8342-7fe654c59737.png)

Base
![image](https://user-images.githubusercontent.com/125171826/218293004-3f14aa73-abdd-41e2-8692-4abc29d773e6.png)

How do you think such impact on the comparison with other models and the reproducibility of results?

**Comment by MenghaoGuo** [comment]
Yes, it depends on benchmark. In all dataset, ADE20K dataset has the largest variance. If you want to get stable result, trying to use cityscapes.
